grantnav/frontend/templates/help.html

```diff
@@ -0,0 +1,305 @@
+          </tr>
+          <tr>
+            <td>{% trans "Recipient District" %}</td>
+            <td>{% trans "This filter is derived from 360Giving data by GrantNav" %}</td>
+            <td>{% blocktrans %}See also: <a href="#location_data">GrantNav and location data</a>{% endblocktrans %}</a></td>
+          </tr>
+          <tr>
+            <td>{% trans "Funding Organisations" %}</td>
+            <td>{% trans "Funding Org:Name" %}</td>
+            <td>{% trans "Values are taken from the data and will change if the funder name changes." %}</td>
+          </tr>
+          <tr>
+            <td>{% trans "Recipient Organisations" %}</td>
+            <td>{% trans "Recipient Org:Name" %}</td>
+            <td>{% blocktrans %}See also: <a href="#charity_names">GrantNav and charity names</a>{% endblocktrans %}</a></td>
+          </tr>
+        </tbody>
+      </table>
+    </div>
+  </div>
+
+  <h2 id="additional_data">{% trans "Additional data and GrantNav" %}</h2>
+  <p>{% blocktrans %}GrantNav uses <a href="/datasets/">grants data</a> that is being published to the <a href="http://standard.threesixtygiving.org/en/latest/">360Giving Standard</a>. It is updated on a monthly basis. When the data is imported, we undertake
+    two processes that mean we can provide additional data.{% endblocktrans %}</p>
+
+  <h3 id="location_data">{% trans "GrantNav and location data" %}</h3>
+  <p>{% blocktrans %}When we import this data, we look at the Location information associated with the Recipient Organisation. This data can vary across publishers.{% endblocktrans %}</p>
+
+  <h4>{% trans "When a Recipient Org:Postal Code is provided" %}</h4>
+  <p>{% blocktrans %}We use the postcode to add Ward, District and Region <a href="/datasets/">codes</a> to the data. This is used in the relevant filters in GrantNav, and also Ward, District and Region names are included in the download files.{% endblocktrans %}</p>
+
+  <h4>{% trans "When a Recipient Org:Postal Code is missing" %}</h4>
+  <p>{% blocktrans %}If other types of administrative geography are included in the data (such as District or Ward codes) then we attempt to match it to relevant areas. Again, these are used in the filters and the names and codes are included in the download files.{% endblocktrans %}</p>
+
+  <h3 id="charity_names">{% trans "GrantNav and charity names" %}</h3>
+  <p>{% blocktrans %}It’s useful for people to see multiple grants associated with a single organisation together in GrantNav. To do this, we rely on publishers including the relevant company, charity or other registration number in the Recipient Org:Identifier
+    field.{% endblocktrans %}</p>
+
+  <p>{% blocktrans %}When we import data into GrantNav, we aim to match these identifiers, in order to provide pages that bring together grants from different publishers, even if they spell the recipient names differently.{% endblocktrans %}</p>
```

Comment:
However, we can still find the same org name with different identifiers. We need to give an explnantion about that as well.

Reply:
I don't think we need to. The freetext search (and advanced search techniques) already cover how we can find a funder/recipient by name, whatever the identifier. It even says this a couple of lines below this one:
> However, we also maintain the original name from the publisher, and use that in the grant page and freetext search.

This part of the page is specifically about how Charity Register data is used in the import, and why.